### DISCUSSION TITLE
Permission to add JobRunner as a feature in Micronaut Launch.

### DISCUSSION THREAD

**Discussion by sdelamo** [jobrunr/jobrunr#792] [General]
Are you ok if we add JobRunner to [Micronaut Launch](https://launch.micronaut.io) as a feature?

[I created an issue for it ](https://github.com/micronaut-projects/micronaut-starter/issues/1887)

**Comment by rdehuyss** [comment]
Hi @sdelamo ! Yes, please go for it. Thanks!

Small remark: JobRunr has a different spelling than you mentioned in both issues.

**Reply by sdelamo** [reply]
> Small remark: JobRunr has a different spelling than you mentioned in both issues.

🤦🏻

**Reply by rdehuyss** [reply]
No worries! 😊
